bootcamp-milestone-2/src/components/PortfolioCard.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import Image from "next/image";
+import styles from "./PortfolioCard.module.css";
+
+export default function PortfolioCard(props: Portfolio) {
+  return (
+    <Link className={styles.link} href={`/portfolio/${props.slug}`}>
+      <div className={styles.project}>
+        <Image
+          src={props.image}
+          alt="Screenshot of this site's index.html page"
+          width="500"
+          height="317"
+        />
+        <div className={styles.projectDetails}>
+          <h2 className="projectName">
+            <strong>{props.projectName}</strong>
```

Comment:
This here is suspicious and might be related to your hydration error. You can change the font weight via CSS instead of a `strong` tag